src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -117,6 +117,70 @@ def open_browser(self, url, browser='firefox', alias=None,
         uses. Notice that prior to SeleniumLibrary 3.0, the library
         contained its own profile that was used by default.
 
+        Optional ``options`` argument allows to define browser specific
+        Selenium options. Example for Chrome, the ``options`` argument
+        allows defining the following
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_chrome/selenium.webdriver.chrome.options.html#selenium.webdriver.chrome.options.Options|methods and attributes]
+        and for Firefox these
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_firefox/selenium.webdriver.firefox.options.html?highlight=firefox#selenium.webdriver.firefox.options.Options|methods and attributes]
+        are available. Please note that not all browsers supported by the
+        SeleniumLibrary have Selenium options available, please consult
+        the Selenium documentation which browser do support the Selenium
+        options. Selenium options are also supported, when ``remote_url``
+        argument is used.
+
+        The SeleniumLibrary ``options`` argument accepts Selenium
+        options in three different formats 1) As a sting, 2) List containing
+        dictionaries 3) Python object which is an instance of the
+        Selenium options object.
+
+        The sting format is allows to define Selenium options methods
```

Comment:
- `sting` -> `string`
- `is allows` -> `allows`

Reply:
Fixed.